larsim/MCCheater/PhotonBackTracker.tcc

```diff
@@ -17,45 +17,52 @@ namespace cheat{
     void PhotonBackTracker::PrepOpDetBTRs(Evt const& evt)
     {
       if(this->BTRsReady()){ return;}
-      auto const& btrHandle = evt.template getValidHandle < std::vector < sim::OpDetBacktrackerRecord > > (fG4ModuleLabel);
-      //      if(btrHandle.failedToGet()){
-      /*  mf::LogWarning("PhotonBackTracker") << "failed to get handle to     simb::MCParticle from "
-       *              << fG4ModuleLabel
-       *                          << ", return";*/ //This is now silent as it is expected to    happen every generation run. It is also temporary while we wait for
-      /*if( 0 ){ return;} //Insert check for DivRecs here, or don't use validHandle below.
-        auto const& divrecHandle = evt.template getValidHandle <std::vector<sim::OpDetDivRec>>(fWavLabel);
-        if(divrecHandle.failedToGet()){
-        return;
-        }*/
+      const std::vector<art::InputTag> G4ModuleLabels = (fG4ModuleLabels.empty()) ?
+        std::vector{fG4ModuleLabel} : fG4ModuleLabels;
```

Comment:
This is a little bit dangerous, especially since a user might explicitly specify `G4ModuleLabel` while something upstream specifies `G4ModuleLabels`. I know that's not how it is now, but it would create an invisible failure that would be very hard to track down. Could you add a check which throws a configuration error if both `G4ModuleLabel` and `G4ModuleLabels` are non-empty (or both empty, for that matter)?